### ISSUE TITLE
--standalone generates file in incorrect PWD

### ISSUE THREAD

**Issue by ismell** [bundler/bundler#2048]
Hello,
I ran the following

```
bundle install --gemfile=src/main/webapp/WEB-INF/Gemfile --path ./target/bundler/ --standalone
```

and it generated the setup file in the following location `target/bundler/bundler/setup.rb` while it installed the gems in `src/main/webapp/WEB-INF/target/bundler/`. So it assumed that the --path was relative to the Gemfile instead of the PWD. It also created the .bundle/config in the WEB-INF folder.

Personally I think the --path option should be relative to the PWD but it seems to be relative to the Gemfile. So I think we should just make the bundler/setup.rb also relative to the Gemfile's path.

**Comment by LynnCo** [comment]
So it looks like this commit https://github.com/bundler/bundler/commit/1447f0fda6af2ea750f6ef9d54ae4d9bb573387b still needs to be added to the 2.0 branch https://github.com/bundler/bundler/tree/2-0-dev

**Comment by LynnCo** [comment]
- which should wait until 2.0 is brought back up to date with master https://github.com/bundler/bundler/pull/4564

**Comment by LynnCo** [comment]
Alright this one is ready for something to take 👍

**Comment by agis** [comment]
@denniss Nope, I hadn't had the time to work on this. Feel free to take a stab.

**Comment by segiddins** [comment]
This should probably be implemented with a feature flag against master, if anyone is interested in tackling this!

**Comment by indirect** [comment]
Closed by #5803.
